### ISSUE TITLE
How to GET an object of type String?

### ISSUE THREAD

**Issue by rafalimaz** [iwpnd/pyle38#68]
Hi, I am having some trouble with get command:
```
async def main():
     await tile38.set("car",'car1').string('a').exec()
     await tile38.get("car",'car1').asObject()

asyncio.run(main())
````

Then I got following error:
``` 
Traceback (most recent call last):
  File "producer.py", line 26, in <module>
    asyncio.run(main())
  File "....\AppData\Local\Programs\Python\Python38\lib\asyncio\runners.py", line 44, in run
    return loop.run_until_complete(main)
  File ".....\AppData\Local\Programs\Python\Python38\lib\asyncio\base_events.py", line 616, in run_until_complete
    return future.result()
  File "producer.py", line 23, in main
    print(await tile38.get("car", 'car1').asObject())
  File "....\AppData\Local\Programs\Python\Python38\lib\site-packages\pyle38\commands\get.py", line 57, in asObject
    return ObjectResponse(**(await self.exec()))
  File "pydantic\main.py", line 406, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for ObjectResponse
object
  value is not a valid dict (type=type_error.dict)

**Comment by iwpnd** [comment]
Hi! 👋 

You can either store a point OR a string, but not both. Check out the Tile38 docs for how to use strings. https://tile38.com/commands/set#strings

**Comment by rafalimaz** [comment]
Yeah,  I know, but the problem is when I try to get the string as object. How to get a string?

**Comment by iwpnd** [comment]
You use the [search](https://tile38.com/commands/search) method for strings. Get should work too here, but doesn't because I'm not expecting strings in the return value right now and it errors out.

```python
import asyncio
from pyle38.tile38 import Tile38

async def main():
    tile38 = Tile38(url="redis://localhost:9851")

    await tile38.set("fleet", "car").point(1, 1).exec()
    await tile38.set("fleet", "car:driver").string("John").exec()

    car = await tile38.get("fleet", "car").asObject()
    print(car.dict())

    driver = await tile38.search("fleet").match("John").asStringObjects()
    print(driver.dict())

    # this should work, but doesn't, will fix
    driver = await tile38.get("fleet", "car:driver").asObject()
    print(driver.dict())


asyncio.run(main())
```

I will fix this on sunday and update doccs accordingly, thanks for reporting. :)

**Comment by rafalimaz** [comment]
Thanks for the answer and for the tip. Unfortunately I am saving encrypted strings so I cannot search them. So, I will wait for the fix on the get().asObject() method. If you need some help just contact me.

**Comment by iwpnd** [comment]
This should be fixed in 0.5.1

```python
import asyncio
from pyle38 import Tile38

async def main():
    tile38 = Tile38(url="redis://localhost:9851", follower_url="redis://localhost:9851")
    await tile38.set("fleet", "truck1:driver").string("John").exec()

    response = await tile38.get("fleet","truck1:driver").asStringObject()
    assert response.ok

    print(response.dict())

asyncio.run(main())

> {
    "ok": True,
    "elapsed": "48.8µs",
    "object": "John" 
}
```

**Comment by iwpnd** [comment]
@rafalimaz  This should be fixed in 0.5.1. Thank you for reporting. :)

```python
import asyncio
from pyle38 import Tile38

async def main():
    tile38 = Tile38(url="redis://localhost:9851", follower_url="redis://localhost:9851")
    await tile38.set("fleet", "truck1:driver").string("John").exec()

    response = await tile38.get("fleet","truck1:driver").asStringObject()
    assert response.ok

    print(response.dict())

asyncio.run(main())

> {
    "ok": True,
    "elapsed": "48.8µs",
    "object": "John" 
}
```
